src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -369,6 +369,7 @@ protected virtual ImmutableArray<NamedTypeSymbol> GetAllInterfaces()
         /// TypeSymbol.Interfaces as the source of edge data, which has had cycles and infinitely
         /// long dependency cycles removed. Consequently, it is possible (and we do) use the
         /// simplest version of Tarjan's topological sorting algorithm.
+        /// Returned interfaces are unique with regards to tuple names.
```

Comment:
It is poorly phrased. To take an example, this return cannot both `I<(int a, int b)>` and `I<(int c, int d)>` (only one will be returned).
Initially, I had "Returned interfaces are unique ignoring tuple names", but that is even less clear.

I'll improve the wording.